js/jest.config.js

```diff
@@ -45,7 +45,8 @@ module.exports = {
       "^.+\\.tsx?$": "ts-jest"
     },
     "transformIgnorePatterns": [
-      "/node_modules/(?!web-stream-tools).+\\.js$"
+      "/node_modules/(?!web-stream-tools).+\\.js$",
+      ".*/cjs/.*\\.js$"
```

Comment:
Without this, I am getting this strange error when I run `yarn gulp test:es5:cjs`?

```
ts-jest[ts-compiler] (WARN) Unable to process '/Users/dominik/Code/arrow/js/targets/es5/cjs/io/adapters.js', falling back to original file content. You can also configure Jest config option `transformIgnorePatterns` to ignore /Users/dominik/Code/arrow/js/targets/es5/cjs/io/adapters.js from transformation or make sure that `outDir` in your tsconfig is neither `''` or `'.'`
```

Any idea what's causing it?

Reply:
It looks like our setup is unhappy about having ts files right next to js files. I think I can redo this on top of #10371.